kanidmd/src/lib/actors/v1_write.rs

```diff
@@ -786,7 +778,7 @@ impl Handler<PurgeRecycledEvent> for QueryServerWriteV1 {
 
     fn handle(&mut self, msg: PurgeRecycledEvent, _: &mut Self::Context) -> Self::Result {
         let mut audit = AuditScope::new("purge recycled");
-        let res = audit_segment!(&mut audit, || {
+        audit_segment!(&mut audit, || {
```

Comment:
And here?

Reply:
Same. Only `()` can be returned.

Reply:
Okay I looked and I can see it now, good spot. Thanks!